### PR TITLE
CDITCK-532 add tests for request context activation.

### PR DESCRIPTION
**Pull request by tremes**

### CODE REVIEW

File: impl/src/main/java/org/jboss/cdi/tck/tests/se/context/activation/interceptor/MethodInterceptorContextActivator.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.cdi.tck.tests.se.context.activation.interceptor;
+
+import javax.enterprise.context.control.ActivateRequestContext;
+import javax.inject.Inject;
+
+@ActivateRequestContext
```
Comment (manovotn) [inline impl/src/main/java/org/jboss/cdi/tck/tests/se/context/activation/interceptor/MethodInterceptorContextActivator.java]: If you want a method interceptor, shouldn't this annotation be only on the given method?

File: impl/src/main/java/org/jboss/cdi/tck/tests/se/context/activation/interceptor/BeforeActivationInterceptor.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    BeanManager beanManager;
+
+    public static AtomicBoolean isRequestContextActive = new AtomicBoolean(false);
+
+    @AroundInvoke
+    public Object intercept(InvocationContext ctx) throws Exception {
+
+        try {
+            beanManager.getContext(RequestScoped.class).isActive();
+            isRequestContextActive.set(true);
```
Comment (manovotn) [inline impl/src/main/java/org/jboss/cdi/tck/tests/se/context/activation/interceptor/BeforeActivationInterceptor.java]: I would probably merge the two lines into one:
`isRequestContextActive.set(beanManager.getContext(RequestScoped.class).isActive());`

Same goes for the other interceptor (`AfterActivationInterceptor`).
Though its just my preference I guess :)

File: impl/src/main/java/org/jboss/cdi/tck/tests/se/context/activation/interceptor/MethodInterceptorContextActivator.java (hunk 1 above)
Comment (tremes) [inline impl/src/main/java/org/jboss/cdi/tck/tests/se/context/activation/interceptor/MethodInterceptorContextActivator.java]: Good catch. Thanks

File: impl/src/main/java/org/jboss/cdi/tck/tests/se/context/activation/interceptor/BeforeActivationInterceptor.java (hunk 2 above)
Comment (tremes) [inline impl/src/main/java/org/jboss/cdi/tck/tests/se/context/activation/interceptor/BeforeActivationInterceptor.java]: The way you suggested could be more readable.

File: impl/src/main/java/org/jboss/cdi/tck/tests/se/context/activation/interceptor/MethodInterceptorContextActivator.java (hunk 1 above)
Comment (mkouba) [inline impl/src/main/java/org/jboss/cdi/tck/tests/se/context/activation/interceptor/MethodInterceptorContextActivator.java]: Copy and paste from `ClassInterceptorContextActivator` ;-)

**Comment by tremes** [comment]
Merged.

**Pull request closed without merging**